standalone/node/src/command.rs

```diff
@@ -255,34 +254,6 @@ pub fn run() -> sc_cli::Result<()> {
                 Ok((cmd.run(client, backend, Some(aux_revert)), task_manager))
             })
         }
-        #[cfg(feature = "try-runtime")]
```

Comment:
Is `try-runtime` completely removed?

Reply:
Moved to https://github.com/paritytech/try-runtime-cli